communication/README.md

```diff
@@ -1,106 +1,193 @@
 # Communication
 
-The Kubernetes community abides by the [Kubernetes code of conduct].  Here is an excerpt:
+The Kubernetes community abides by the [Kubernetes code of conduct] on all of
+communication platforms that we moderate listed below with noted exceptions.  
+Here is an excerpt from the code of conduct:
 
 > _As contributors and maintainers of this project, and in the interest
 > of fostering an open and welcoming community, we pledge to respect
 > all people who contribute through reporting issues, posting feature
 > requests, updating documentation, submitting pull requests or patches,
 > and other activities._
 
-## SIGs
+## Purpose of this doc
 
-Kubernetes encompasses many projects, organized into [SIGs](/sig-list.md).
-Some communication has moved into SIG-specific channels - see
-a given SIG subdirectory for details.
+A detailed list of upstream communication platforms and resources for contributors.
+Since upstream contributors are generally consumers, many of our channels intertwine.
+See [Misc Community Resources] for more end user/troubleshooting targeted paths.
+
+## Community Groups
+
+Kubernetes encompasses many projects, organized into [community groups].
+The upstream communication flows through those channels, most
+notably in the Special Interest Groups [SIGs] that own the docs and
+codebases. Their communication channels may include mailing lists, slack channels,
+zoom meetings, meeting agenda/notes, and can be found on their READMEs and on the
+community groups/[SIGs] page.
+
+You can actively or passively participate in one of the following ways:
+- the community groups public meeting(s) listed in the above community groups page
+- every Thursday at our [weekly community meeting] over [zoom] at [10am US Pacific Time]
+- intro sessions at KubeCon/CloudNativeCon live or recordings on YouTube[need a link here]
```

Comment:
CNCF Channel link: https://www.youtube.com/channel/UCvqbFHwN-nwalWPjPUKpvTA

Reply:
A playlist would be ideal, but the CNCF channel link should do for now.